lib/go-tc/servers.go

```diff
@@ -49,6 +64,157 @@ type ServersV1DetailResponse struct {
 	Alerts
 }
 
+// ServerIpAddress is the data associated with a server's interface's IP address.
+type ServerIpAddress struct {
+	Address        string  `json:"address" db:"address"`
+	Gateway        *string `json:"gateway" db:"gateway"`
+	ServiceAddress bool    `json:"serviceAddress" db:"service_address"`
+}
+
+// ServerInterfaceInfo is the data associated with a server's interface.
+type ServerInterfaceInfo struct {
+	IPAddresses  []ServerIpAddress `json:"ipAddresses" db:"ip_addresses"`
+	MaxBandwidth *uint64           `json:"maxBandwidth" db:"max_bandwidth"`
+	Monitor      bool              `json:"monitor" db:"monitor"`
+	MTU          *uint64           `json:"mtu" db:"mtu"`
+	Name         string            `json:"name" db:"name"`
+}
+
+// Value implements the driver.Valuer interface
+// marshals struct to json to pass back as a json.RawMessage
+func (sii *ServerInterfaceInfo) Value() (driver.Value, error) {
+	b, err := json.Marshal(sii)
+	return b, err
+}
+
+// Scan implements the sql.Scanner interface
+// expects json.RawMessage and unmarshals to a deliveryservice struct
+func (sii *ServerInterfaceInfo) Scan(src interface{}) error {
+	b, ok := src.([]byte)
+	if !ok {
+		return fmt.Errorf("expected deliveryservice in byte array form; got %T", src)
+	}
+
+	return json.Unmarshal([]byte(b), sii)
+}
+
+// LegacyInterfaceDetails is the details for interfaces on servers for API v1 and v2.
+type LegacyInterfaceDetails struct {
+	InterfaceMtu  *int    `json:"interfaceMtu" db:"interface_mtu"`
```

Comment:
`Mtu` should be capitalized

Reply:
It should, but it can't be for backward-compatibility. This isn't a new property, it's moved from the old struct.